core/src/main/java/feast/core/service/SpecService.java

```diff
@@ -682,19 +682,28 @@ public ApplyFeatureTableResponse applyFeatureTable(ApplyFeatureTableRequest requ
 
   /**
    * List the FeatureTables matching the filter in the given filter. Scopes down listing to project
-   * if specified, the default project otherwise.
+   * if specified, otherwise all FeatureTables will be returned.
    *
    * @param filter Filter containing the desired project and labels
    * @return ListFeatureTablesResponse with list of FeatureTables found matching the filter
    */
   @Transactional
   public ListFeatureTablesResponse listFeatureTables(ListFeatureTablesRequest.Filter filter) {
-    String projectName = resolveProjectName(filter.getProject());
+    String projectName = filter.getProject();
     Map<String, String> labelsFilter = filter.getLabelsMap();
 
-    checkValidCharacters(projectName, "project");
+    if (projectName.isEmpty()) {
+      projectName = "*";
```

Comment:
We do we persist with these wildcard patterns?

Reply:
Since we're keeping FeatureSets functionality inside the Serving implementation, we need to manage the use of wildcard in application.yml when starting Serving service. An example would be our integration tests which uses [these settings](https://github.com/feast-dev/feast/blob/master/serving/src/test/java/feast/serving/it/BaseAuthIT.java#L75).

Reply:
It's not clear how FeatureSets and FeatureTables are connected here. Aren't they completely separate?

Reply:
They are but we're using the same set of subscriptions from the application.yml.

Reply:
Why do we need a subscription for FeatureTables again?

Reply:
So we're doing away with using subscriptions from application.yml like the logic [here](https://github.com/feast-dev/feast/blob/master/serving/src/main/java/feast/serving/specs/CachedSpecService.java#L235) and directly list all tables?

Reply:
We will only have a single online store, or a single stream. So I dont think we need subs for `feature tables` any more. 

What you have linked to is for `feature sets`. We still need that right.

Reply:
Yup we still need it for `feature sets`. Okay, I'll remove the `*` and use `%` directly then.

Reply:
Why do we need to use `%`?